RELEASE.md

```diff
@@ -33,12 +33,20 @@ $ export GH=/path/to/gh
 
 ## Pre-release tests
 
-Verify the references to the internal Actions by manually running the [release workflow](https://github.com/slsa-framework/slsa-github-generator/actions/workflows/release.yml). Ensure this workflow succeeds. If the references are still `@main`, update them with the following command:
+Verify the references to the internal Actions by manually running the [release workflow](https://github.com/slsa-framework/slsa-github-generator/actions/workflows/release.yml). Ensure this workflow succeeds.
+
+If workflows still refer to `@main`, update them with the following command:
 
 ```shell
 find .github/workflows/ -name '*.yaml' -o -name '*.yml' | xargs sed -i 's/uses: slsa-framework\/slsa-github-generator\/\.github\/actions\/\(.*\)@main*/uses: slsa-framework\/slsa-github-generator\/.github\/actions\/\1@_YOUR_RELEASE_TAG_/'
 ```
 
+Likewise, if documentation refers to a previous release, update it with the following command:
+
+```shell
+$ find . -name "*.md" -exec sed -i "s/_PREVIOUS_RELEASE_TAG_/_YOUR_RELEASE_TAG_/g" {} +
```

Comment:
Maybe (not sure) we could add a PREVIOUS_RELEASE_TAG as env variables, so that here we could say:
```shell
$ find . -name "*.md" -exec sed -i "s/$PREVIOUS_RELEASE_TAG/$RELEASE_TAG/g" {} +
```

Reply:
Not sure I understand. This is run locally as a throwaway command, no? We wouldn't even have access to the Action's `$RELEASE_TAG` variable.

Reply:
we run these commands on our local machines. When doing the release process, we set up a series of env variables (https://github.com/slsa-framework/slsa-github-generator/blob/3c5599ba46a13926b6daf4f67202a28c4d6183d3/RELEASE.md#prerequisites). I was suggesting we add another one `PREVIOUS_RELEASE_TAG` to avoid mistakes.

Reply:
Ah yes, of course. 🤦‍♂️

But we'd then need both RELEASE_TAG and PREVIOUS_RELEASE_TAG, no?

Reply:
yes. REQLSE_TAG is already set, it's called BUILDER_TAG. So maybe call the previous one BUILDER_PREVIOUS_TAG?

Reply:
Done. See if my implementation is ok.

Reply:
If of interest, I could also simplify the `sed` commands by using another separator, i.e.:

```shell
sed -i "s/uses: slsa-framework\/slsa-github-generator\/\.github\/actions\/\(.*\)@main*/uses: slsa-framework\/slsa-github-generator\/.github\/actions\/\1@$BUILDER_TAG/"

# becomes

sed -i "s#uses: slsa-framework/slsa-github-generator/\.github/actions/\(.*\)@main*#uses: slsa-framework/slsa-github-generator/.github/actions/\1@$BUILDER_TAG#"
```